test/utils/devices/temperature_control_test.py

```diff
@@ -73,15 +55,6 @@ def test_temperature_control_last_temperature():
     temperature_controller.get_last_temperature()
 
 
-def test_temperature_control_activate():
```

Comment:
How can you test this in the new code base?

Reply:
This test has been implemented